lib/docker/testhelpers.go

```diff
@@ -0,0 +1,84 @@
+package docker
+
+import (
+	"context"
+
+	"github.com/gravitational/gravity/lib/utils"
+
+	dockerapi "github.com/fsouza/go-dockerclient"
+)
+
+// MockDocker is a mock Docker client implementation used in unit tests.
+type MockDocker struct{}
```

Comment:
Would it make sense to use the real docker? We run tests in an environment that has docker access in at least one of the branches (might be master). What is the benefit of this mock?

Reply:
Just mocks it in the new vendoring test I added where we don't need Docker.